proto/lbm/token/v1/tx.proto

```diff
@@ -60,8 +43,15 @@ service Msg {
   // Since: 0.46.0 (finschia)
   rpc RevokeOperator(MsgRevokeOperator) returns (MsgRevokeOperatorResponse);
```

Comment:
Does `RevokeOperator` keep remaining, not removing? It is new added function.

Reply:
The planning team added it to the spec, so we cannot remove it. Or, we must contact them to remove it here.